src/main/java/testapp/adapter/memory/InMemoryUserRepository.java

```diff
@@ -26,9 +25,9 @@ public InMemoryUserRepository() {
   }
 
   @Override
-  public User findUser(Integer userId) {
+  public User findUser(Integer facultyNumber) {
     for (User user : userList) {
-      if (user.getId().equals(userId)) {
+      if (user.getFacultyNumber().equals(facultyNumber)) {
```

Comment:
they are, but it says facultyNumber for one, how do we know that in other facultys do not exist the same Numbers, and if they dont' what about other Universities faculties?